src/translations/en.json

```diff
@@ -3396,8 +3396,7 @@
       "addon_stopped": "Add-on is stopped",
       "addons": "Add-ons",
       "no_addons": "You don't have any add-ons installed yet. Head over to the add-on store to get started!",
-      "update_available": "Update pending {icon}",
-      "updates_available": "Updates pending {icon}"
+      "update_available": "{count, plural,\n  one {Update}\n  other {{count} Updates}\n} pending {icon}",
```

Comment:
I think my {count} should be #... If you want to display the number of updated that is...
```
      "update_available": "{count, plural,\n  one {Update}\n  other {# Updates}\n} pending {icon}",

```